meshio/nastran/_nastran.py

```diff
@@ -192,7 +192,82 @@ def add_cell(cell, cell_type, cell_ref, keyword_prev):
     return mesh
 
 
-def write(filename, mesh):
+def print_scientific(value, length=16):
```

Comment:
This method doesn't print, it returns. Any why "shouldnt it be called"? Also, why not using `{:.16e}`?

Reply:
This is a method proposed by @SteveDoyle2, see comments above. The method can make full use of all 16 characters. It checks the sign, leaves out "e" for exponent, ...

Reply:
Yes, the method name can be changed, eg. to `_scientific_format()`. Or `_float_nastran()`, because it is a "inverse" method to `_nastran_float()` employed by the reader.

Reply:
`float_to_nastran_string` sounds good. Please rename the inverse, too. The doc string needs a couple of illustrating examples which floats are converted to what.

Reply:
Okay, renamed. Examples of float to string conversion are in the docstring of  `_float_to_nastran_string()`.